authors/apps/articles/tests/test_rate_article.py

```diff
@@ -0,0 +1,81 @@
+from django.urls import reverse
+from rest_framework import status
+from ...authentication.tests import base_class
+from .test_data import test_rate_article_data
+from ..models import Rating, Article
+
+
+class TestRatingModel(base_class.BaseTest):
+    """
+    This class tests the Rating View
+    """
+
+    def setUp(self):
+        super().setUp()
+        self.user_author = self.activated_user()
+        self.client.force_authenticate(user=self.user_author)
+        self.article = self.create_article(self.user_author)
+        self.url_rate_article = reverse('articles:article-rates',
+                                        kwargs={"slug": self.article.slug})
+        self.url_articles = reverse('articles:articles')
+
+    """unit tests for module under rating articles"""
+
+    def test_rate_article_author(self):
+        """
+        Test rating an article fails when author tries to rate their
+        own article
+        """
+        response = self.client.post(self.url_rate_article,
+                                    data=test_rate_article_data.valid_rate_data,
+                                    format='json')
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+        self.assertIn("Author can not rate their own article",
+                      response.data.get('errors'))
+
+    """integration tests for all modules under rating articles"""
```

Comment:
@missvicki this docstring also I think it's miss placed it should be at the top of the class

Reply:
this is supposed to indicate where integration tests start. I think it is best if i made it a comment. So as to remove the confusion